api/Window.json

```diff
@@ -153,264 +228,2085 @@
+              "version_added": "47"
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "resizeBy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/resizeBy",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true,
+              "notes": "Since Firefox 7, it's no longer possible for a web site to change the default size of a window in a browser, according to the <a href='https://bugzil.la/565541#c24'>following rules</a>:<ol><li>You can't resize a window or tab that wasn’t created by <code>window.open</code>.</li><li>You can't resize a window or tab when it’s in a window with more than one tab.</li></ol>"
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "resizeTo": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/resizeTo",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true,
+              "notes": "Since Firefox 7, it's no longer possible for a web site to change the default size of a window in a browser, according to the <a href='https://bugzil.la/565541#c24'>following rules</a>:<ol><li>You can't resize a window or tab that wasn’t created by <code>window.open</code>.</li><li>You can't resize a window or tab when it’s in a window with more than one tab.</li></ol>"
+            },
```

Comment:
Same comment: It's my understanding that you can only use `<a>` and `<code>` in ntes (https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#notes).